pkg/apis/kubeone/v1beta2/types.go

```diff
@@ -61,6 +61,15 @@ type KubeOneCluster struct {
 	SystemPackages *SystemPackages `json:"systemPackages,omitempty"`
 	// RegistryConfiguration configures how Docker images are pulled from an image registry
 	RegistryConfiguration *RegistryConfiguration `json:"registryConfiguration,omitempty"`
+	// LoggingConfig configures the Kubelet's log configuration
+	LoggingConfig LoggingConfig `json:"loggingConfig,omitempty"`
+}
+
+// LoggingConfig
```

Comment:
Please also extend this comment like in the internal API:

```suggestion
// LoggingConfig configures the Kubelet's log rotation
```